kitsune/sumo/middleware.py

```diff
@@ -36,17 +36,17 @@
 class EnforceHostIPMiddleware(EnforceHostMiddleware):
     """Modify the `EnforceHostMiddleware` to allow IP addresses"""
 
-    def process_request(self, request):
```

Comment:
Why are you changing this here?

Reply:
It was failing tests with this: `AttributeError: 'super' object has no attribute 'process_request'` - while cleaning up the super() call it seemed to make sense to modernize this middleware to use `__call__` since we don't use 'MIDDLEWARE_CLASSES'.

Reply:
definitely a nit - instead of blocking this, let's create an issue to update all instances